qutebrowser/config/configdata.yml

```diff
@@ -43,6 +43,11 @@ ignore_case:
   default: smart
   desc: Find text on a page case-insensitively.
 
+incremental_search:
+  type: Bool
+  default: False
```

Comment:
I think it's fine to default this to True, so people actually discover that it exists :wink: